src/layout/RepeatingGroup/index.tsx

```diff
@@ -93,6 +93,22 @@ export class RepeatingGroup extends RepeatingGroupDef implements ValidateCompone
     return validations;
   }
 
+  /**
+   * Repeating group has its own minCount property, so if set, we should filter out the minItems validation from schema.
+   */
+  schemaMinItemsFilter(validation: BaseValidation): boolean {
```

Comment:
This one could probably be `private`. I guess we don't expect it to be used outside the class itself?

Reply:
good point